tests/pyccel/test_pyccel.py

```diff
@@ -652,6 +652,10 @@ def test_arrays_view(language):
             [int] * 6 + [int] * 10
     pyccel_test("scripts/arrays_view.py", language=language, output_dtype=types)
 
+def test_return_numpy_arrays(language):
+    types = [int] * 5 + [(type(np.array),) * 2]
```

Comment:
These types concern the values that will be read from the terminal. It is important so that the test function knows how to parse the output (ints are parsed differently to floats and differently to strings). You cannot specify `type(np.array)` here as this does not tell it how to read it. Here you should have: `int` (all the values are ints so you shouldn't need to create a list of the right size)